web/src/components/PluginEditor.vue

```diff
@@ -70,13 +70,25 @@ export default {
   data() {
     return {
       codeValue: '',
+      editor: null
     }
   },
   created(){
     this.store = this.$root.$data.store
   },
   watch: {
-
+    // window: {
```

Comment:
Do we want to keep the commented code?

Reply:
I see, I should have removed those. I will do it in another PR then.